src/main.cpp

```diff
@@ -13,11 +13,20 @@
 #define SERIAL_BITS SERIAL_8N1
 #endif
 
+
+// Uncomment and fill in the next 4 lines to use a static IP address
+// #define STATIC_IP
+// IPAddress ip(192, 168, 0, 253);
+// IPAddress gateway(192, 168, 0, 1);
+// IPAddress subnet(255, 255, 255, 0);
+
 WiFiServer server(23);
 WiFiClient current_client;
 
 #define IDEAL_PACKET_SIZE 1024
 void setup() {
+	pinMode(LED_BUILTIN, OUTPUT);
```

Comment:
nitpick: the indentation is tabs for these 2 lines, not spaces like the rest of the file.